addons/compat_ws/CfgEventHandlers.hpp

```diff
@@ -0,0 +1,13 @@
+class Extended_InitPost_EventHandlers {
+    class UAV_02_Base_lxWS {
+        class ADDON {
+            init = QUOTE(call EFUNC(cookoff,disableCookoff));
+        };
```

Comment:
I think we just want something like this
```suggestion
            exclude[] = {"UAV_02_IED_Base_lxWS "};
        };
```

Reply:
From what I could gather on the wiki, doesn't `exclude` only apply to single classes and not their children or am I wrong?
I thought it didn't (didn't test it though), so I didn't use it, but if it does exclude children then I'm all for it.